File: Creating the contests/Creating_the_contests.cpp
#include <bits/stdc++.h>
#define ll long long
using namespace std;

ll Creating_the_Contest()
{  ll total_problems;
   cin>>total_problems;
   vector<ll> diff_of_prob;
   
   for(int i = 0;i<total_problems;i++)
   {   ll A;
       cin>>A;
       diff_of_prob.push_back(A);
   }
   
   ll max_length = 1;
   ll count = 0;
   for(int i = 0; i < total_problems-1;i++)
   {
       if(diff_of_prob[i+1] <= diff_of_prob[i]*2)
           count += 1;
       else
       {
           max_length = max(max_length,count+1);
           count = 0;
       }
   }
   max_length = max(max_length,count+1);
   return (max_length);
   
}


int main() {
	int remained_test_cases = 1 ;
// 	cin>>remained_test_cases;
	while(remained_test_cases > 0)
	{   cout<<Creating_the_Contest()<<"\n";
	    remained_test_cases--;
	}
	return 0;
}

File: Creating the contests/Creating_the_contests.py
def Creating_the_Contest():
  total_problems = int(input())
  diff_of_prob = list(map(int,input().split()))
  max_subset_len = 1
  count_satisfied = 0
  
  for i in range(total_problems-1):
      if diff_of_prob[i+1] <= 2*diff_of_prob[i]:
          count_satisfied += 1 
      else:
          max_subset_len = max(max_subset_len,count_satisfied+1)
          count_satisfied = 0
          
  max_subset_len = max(count_satisfied+1,max_subset_len)
  return max_subset_len
      
       
       
            
reamined_test_cases = 1         
# reamined_test_cases = int(input())
while reamined_test_cases > 0:
    print(Creating_the_Contest())
    reamined_test_cases -= 1 
